File: algorithm/src/main/java/baekjoon/p10950/Main.java
package baekjoon.p10950;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        int[][] inputs = new int[t][2];

        for (int[] input : inputs) {
            input[0] = sc.nextInt();
            input[1] = sc.nextInt();
        }

        for (int ret : solve(inputs)) {
            System.out.println(ret);
        }
    }

    private static int[] solve(int[][] inputs) {
        int[] ret = new int[inputs.length];

        for (int i = 0; i < inputs.length; i++) {
            ret[i] = inputs[i][0] + inputs[i][1];
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p10950/Main.kt
package baekjoon.p10950

fun main(args: Array<String>) {
    val t = readLine()!!.toInt()
    val inputs = Array(t) { IntArray(2) }

    for (input in inputs) {
        val arr = readLine()!!.split(" ")
        input[0] = arr[0].toInt()
        input[1] = arr[1].toInt()
    }

    for (ret in solve(inputs)) {
        println(ret)
    }
}

private fun solve(inputs: Array<IntArray>): IntArray {
    val ret = IntArray(inputs.size)

    for (i in 0 until inputs.size) {
        ret[i] = inputs[i][0] + inputs[i][1]
    }

    return ret
}